cmd/kubeconform/main.go

```diff
@@ -46,6 +49,26 @@ func processResults(cancel context.CancelFunc, o output.Output, validationResult
 	return result
 }
 
+func parseStdin(stdin *os.File) *strings.Reader {
+	var content string
+
+	// Read all stdin lines.
+	scanner := bufio.NewScanner(stdin)
+	for scanner.Scan() {
+		content += fmt.Sprintln(scanner.Text())
+	}
+
+	// Check if the stdin input is ResourceList.
+	rl, err := fn.ParseResourceList([]byte(content))
```

Comment:
This adds support for ResourceLists (I need to read about this) only if passed via stdin? This would not work if I ran kubeconform ./fixtures/krm.yaml? I think we would need some consistency there